### PR TITLE
Add node addr type to pod to host test

### PR DESCRIPTION
**Pull request by darox**
This commit changes the output format of
the `pod-to-host` test cases.

From:
```
Action [allow-all-except-world/pod-to-host/ping-ipv4-1
```
to:
```
Action [allow-all-except-world/pod-to-host/ping-ipv4-internal-ip
```

by utilizing the `field Type v1.NodeAddressType`

With this change, a user can immediately see if the ping to the ExternalIP or InternalIP of the node is failing. Without the need to run `kubectl get po -o wide` separately.

### CODE REVIEW

**Review by thorn3r — APPROVED** [review]

**Comment by squeed** [comment]
@darox there's a golang-ci lint failure; can you take a look?

**Comment by darox** [comment]
> @darox there's a golang-ci lint failure; can you take a look?

Sure. I will have a look.

**Comment by darox** [comment]
> @darox there's a golang-ci lint failure; can you take a look?

fixed

File: connectivity/tests/host.go [hunk 1]
```diff
@@ -42,10 +42,25 @@ func (s *podToHost) Run(ctx context.Context, t *check.Test) {
...
+						}
+					case addr.Type == "ExternalIP":
+						{
+							addrType = "external-ip"
+						}
+					case addr.Type == "Hostname":
+						{
+							addrType = "hostname"
+						}
+					}
```
Comment (tklauser) [inline connectivity/tests/host.go:58]: How about using the `NodeAddressType` constants defined in `k8s.io/api/core/v1`?

additional nit: no need for the curly braces in Go:

```suggestion
					switch {
					case addr.Type == corev1.NodeInternal:
						addrType = "internal-ip"
					case addr.Type == corev1.NodeExternalIP:
						addrType = "external-ip"
					case addr.Type == corev1.NodeHostName:
						addrType = "hostname"
					}
```

Comment (darox) [reply connectivity/tests/host.go]: Good proposal. Thank you.

Comment (tklauser) [reply connectivity/tests/host.go]: Sorry, I forgot to mention you'll also need to add `corev1 k8s.io/core/v1` to the `import` section at the top of the file. Please also squash the second commit into to first one.

**Review by tklauser — APPROVED** [review]
Thanks @darox!

**Pull request merged**